packages/synapse-react-client/src/components/OrientationBanner/OrientationBanner.tsx

```diff
@@ -54,9 +56,10 @@ function OrientationBanner(props: OrientationBannerProps) {
     localStorage.getItem(storageBannerId) === null,
   )
 
-  const theme = useTheme()
-  const defaultSx = {
-    display: { xs: 'none', md: 'unset' },
```

Comment:
Setting this to `unset` ended up replacing the `flex` layout, so we had to remove `unset` somehow